tests/steps/blocks_test.go

```diff
@@ -0,0 +1,117 @@
+package steps
+
+import (
+	"context"
+	"github.com/cucumber/godog"
+	"github.com/make-software/casper-go-sdk/casper"
+	"github.com/make-software/casper-go-sdk/rpc"
+	"github.com/stormeye2000/cspr-sdk-standard-tests-go/tests/utils"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+type _map struct {
+	blockDataNode casper.Block
+	blockDataSdk  rpc.ChainGetBlockResult
+}
+
+var contextMap _map
+var t *testing.T
+
+func TestFeaturesBlocks(t *testing.T) {
+	suite := godog.TestSuite{
```

Comment:
Does this suite need to go into it's own file to allow importing of multiple tests?